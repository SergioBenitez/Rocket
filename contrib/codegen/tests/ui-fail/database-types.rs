extern crate rocket;
#[macro_use] extern crate rocket_contrib;

struct Unknown;

#[database("foo")]
struct A(Unknown);
//~^ ERROR Unknown: rocket_contrib::databases::Poolable

fn main() {  }
