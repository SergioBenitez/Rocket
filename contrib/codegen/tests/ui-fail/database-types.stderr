error[E0277]: the trait bound `Unknown: rocket_contrib::databases::Poolable` is not satisfied
 --> $DIR/database-types.rs:7:10
  |
7 | struct A(Unknown);
  |          ^^^^^^^ the trait `rocket_contrib::databases::Poolable` is not implemented for `Unknown`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0277`.
